home/modules/ROOT/pages/index.adoc

```diff
@@ -6,7 +6,7 @@
 We’d love to hear from you. Feel free to use the btn:[Feedback?] button on any page to share your feedback and concerns.
 
 [.hint]
-Looking for non-documentation resources? 
+Looking for non-documentation resources?
```

Comment:
I'm not sure why git tracked a change here. I didn't change anything on this line so just ignore.